src/logger.js

```diff
@@ -102,40 +114,60 @@ var warn = process.bind(null, levels.WARN);
 var error = process.bind(null, levels.ERROR);
 
 /**
- * Outputs a warning message to the web console.
- * @param {...*} arg Any argument to be logged.
+ * Outputs a deprecation warning to the web console.
```

Comment:
This was the original ask of the ticket pdcl-13979: Turbine was too catty in a production environment because logger.deprecation was allowed to call the console even when setDebug(false) was called. This file now holds up to 100 unique log messages to de-duplicate and ignore. once the buffer fills up, the first entry is ejected to make room for a new call. 

The goal here is if we see 5 unique deprecation messages over and over and over, we log each of them once and never again when `outputEnabled = false`.